lib/dynamic_config/dcdo.rb

```diff
@@ -40,6 +40,8 @@ def frontend_config
       'progress-table-v2-enabled': DCDO.get('progress-table-v2-enabled', false),
       # Whether to show the v1 or v2 progress table by default.
       'progress-table-v2-default-v2': DCDO.get('progress-table-v2-default-v2', false),
+      # Whether the scholarship dropdown is locked on the application dashboard.
+      'scholarship-dropdown-locked': DCDO.get('scholarship-dropdown-locked', true),
```

Comment:
For my own curiosity/learning, can you explain why this one is set to `true` here while the others are marked `false`?

Reply:
The second parameter is a default and I figured we'd want to default to locking this dropdown. Happy to change that if folks think it should default to being unlocked!

Reply:
Cool, thanks! I think what you have is fine since that's the default. This is good to know for future DCDO options too.